### ISSUE TITLE
iOS: On timeline when the feed name is present it should be presented first to VoiceOver

### ISSUE THREAD

**Issue by vincode-io** [Ranchero-Software/NetNewsWire#1584]
When viewing the list of articles in a folder that contains articles from multiple feeds (such as the Today or Unread folders), I would suggest a tweak to the order in which VoiceOver speaks elements of each article on the list.

Currently, VoiceOver speaks information on the article in the following order: title > Teaser > Feed Name > Timestamp.

I would suggest that the name of the feed be the first thing spoken by VoiceOver.
